src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -279,35 +271,25 @@ export const handleEncryptedSubmission: RequestHandler = async (req, res) => {
   }
 
   // Save Responses to Database
-  // TODO(frankchn): Extract S3 upload functionality to a service
   const formData = req.body.encryptedContent
-  const attachmentData = req.body.attachments || {}
   const { verified } = res.locals
-  const attachmentMetadata = new Map()
-  const attachmentUploadPromises = []
-
-  // Object.keys(attachmentData[fieldId].encryptedFile) [ 'submissionPublicKey', 'nonce', 'binary' ]
-  for (const fieldId in attachmentData) {
-    const individualAttachment = JSON.stringify(attachmentData[fieldId])
-
-    const hashStr = crypto
-      .createHash('sha256')
-      .update(individualAttachment)
-      .digest('hex')
-
-    const uploadKey =
-      form._id + '/' + crypto.randomBytes(20).toString('hex') + '/' + hashStr
-
-    attachmentMetadata.set(fieldId, uploadKey)
-    attachmentUploadPromises.push(
-      AwsConfig.s3
-        .upload({
-          Bucket: AwsConfig.attachmentS3Bucket,
-          Key: uploadKey,
-          Body: Buffer.from(individualAttachment),
-        })
-        .promise(),
+  let attachmentMetadata = new Map()
```

Comment:
again, you can use `Map<string, string>` or a type alias to ensure that TypeScript doesn't revert to the default of `Map<any, any>`

Reply:
Changed to `AttachmentMetadata` type

Reply:
ah never mind, typescript doesn't support a straight `new AttachmentMetadata()` thing. changing to `Map<string, string>`